### ISSUE TITLE
exponentially increasing libnvidia-allocator mount points on host

### ISSUE THREAD

**Issue by BryanQuigley** [NVIDIA/nvidia-container-toolkit#697]
This appears to be the same root issue as #660 or #663 but wanted to detail the impact we are seeing from it.
Is there any timeline for the backported fix to be released?

# Our symptoms
When mounting with a specific container image and a shared mount we get it creating an exponentially increasing amount of mounts on the host. 

It actually creates 2^x - 1 number of mount points, so 1, 3 ... .255.. 32767.  Where x is the number of containers causing the issue have run.

When it gets to the higher numbers it starts causing significant system issues due to the very high number of mounts.

The mount point is basically the same as from the other issue - copying here for searchability (although we are on nvidia 560)
```
/dev/md0 on /usr/lib/x86_64-linux-gnu/libnvidia-allocator.so.550.54.15 type ext4 (rw,relatime,nodelalloc,errors=remount-ro)```

**Comment by BryanQuigley** [comment]
@elezar was this what you were seeing too?

**Comment by elezar** [comment]
@BryanQuigley yes, this is the known issue and is triggered when `NVIDIA_DRIVER_CAPABILITIES` includes `graphics` or `display`. It *should* only be triggered if a container is started with a `bidirectional` mount propagation.

We have a fix in place and this will be included in the next patch release. For the time being, please downgrade to a `v1.15.x` version if possible.

**Comment by BryanQuigley** [comment]
Thanks! I see it was released in 1.16.2.
